charts/k8s-kubectl/templates/deployment.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: apps/v1beta2
```

Comment:
@ffalcn I don't think you need the helm charts. As this is an image that is consumed by other pipelines to build and deploy from their respective pipelines?

Reply:
That makes sense, I will remove the charts.